Projects/Features/RootFeature/Project.swift

```diff
@@ -1,12 +1,19 @@
 import ProjectDescription
 import ProjectDescriptionHelpers
+import DependencyPlugin
 
-let project = Project.makeModule(
-    name: "RootFeature",
-    product: .staticFramework,
-    dependencies: [
-        .Project.Features.BaseFeature,
-        .Project.Features.MainTabFeature
+let project = Project.module(
+    name: ModulePaths.Feature.RootFeature.rawValue,
+    targets: [
+        .implements(
+            module: .feature(.RootFeature),
+            product: .staticFramework,
+            spec: .init(
+                resources: ["Resources/**"],
+                dependencies: [.feature(target: .BaseFeature),
+                               .feature(target: .MainTabFeature),
+                               .domain(target: .AppDomain, type: .interface)]
```

Comment:
```suggestion
                dependencies: [
                    .feature(target: .BaseFeature),
                    .feature(target: .MainTabFeature),
                    .domain(target: .AppDomain, type: .interface)
                ]
```
요기 줄바꿈 요렇게 하는거 어떤가요?